internal/pkg/instrumentation/bpf/net/http/client/bpf/probe.bpf.c

```diff
@@ -265,4 +172,73 @@ int uprobe_Transport_roundTrip_Returns(struct pt_regs *ctx) {
 
     bpf_map_delete_elem(&http_events, &key);
     return 0;
-}
\ No newline at end of file
+}
+
+#ifndef NO_HEADER_PROPAGATION
+SEC("uprobe/header_writeSubset")
```

Comment:
nit: add the function signature as a comment (similar to other probes)

Reply:
Done, thanks!